bin/install.ps1

```diff
@@ -19,7 +19,7 @@ if ($env:PLATFORM -eq "x86") {
 
 if ($env:JULIA_VERSION -in 'latest', 'nightly') {
     # TODO: move to nightly to be consistent with Travis and download page
-    $julia_version = [Version]"0.7"
+    $julia_version = [Version]"1.4"
```

Comment:
Not sure why the version here is hardcoded?  Anyways, separate issue.

Reply:
It only affects the choice of commands used to trigger tests, e.g. https://github.com/JuliaCI/Appveyor.jl/blob/178812944acf5b2020040c779423a3d771d3a9b7/bin/install.ps1#L59.

Reply:
Got it.  Had to update it to 1.4 so  versioning uses the correct command for the nightly builds, in the changes made in this PR.